.github/workflows/inreach-parser.yaml

```diff
@@ -0,0 +1,51 @@
+name: Deploy inreach parser
+
+on:
+  push:
+    branches: ['master', 'dev', 'feat/az-function']
+    paths: ['inreach-parser/**']
+  
+env:
+  FUNCTION_APP_NAME: bikejamboree-inreach-parser  # Name of the function
+  PACKAGE_PATH: ./inreach-parser  # Applciation path in repository
+  RESOURCE_GROUP_NAME: zhp-bikejamboree
+
+jobs:
+  deploy:
+    name: Deploy to Azure Functions
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Login to Azure
+        uses: azure/login@v1
+        with:
+          creds: ${{ secrets.AZURE_CREDENTIALS }}
+
+      - name: Validate Bicep
+        id: bicep-validate
+        uses: azure/arm-deploy@v1
+        with:
+          subscriptionId: ${{ vars.AZURE_SUBSCRIPTION_ID }}
+          resourceGroupName: ${{ env.RESOURCE_GROUP_NAME }}
+          template: ./inreach-parser/inreach-parser.bicep
+          deploymentMode: Validate
+          parameters: serverfarm_id=${{ secrets.SERVERFARM_ID }} insights_id=${{ secrets.INSIGHTS_ID }}
+
+      - name: Deploy Bicep
+        id: bicep-deploy
+        if: steps.bicep-validate.outcome == 'success'
+        uses: azure/arm-deploy@v1
+        with:
+          subscriptionId: ${{ vars.AZURE_SUBSCRIPTION_ID }}
+          resourceGroupName: ${{ env.RESOURCE_GROUP_NAME }}
+          template: ./inreach-parser/inreach-parser.bicep
+          deploymentMode: Incremental
+          parameters: serverfarm_id=${{ secrets.SERVERFARM_ID }} insights_id=${{ secrets.INSIGHTS_ID }}
+
+      - name: Deploy function code
+        if: steps.bicep-deploy.outcome == 'success'
```

Comment:
AFAIK _If success_ to domyślny warunek. Trzeba go pisać, jeśli chcemy go jakoś zmodyfikować, ale tam poza tym to chyba się pomija
```suggestion
```